### ISSUE TITLE
Notes from paris

### ISSUE THREAD

**Issue by Hoverbear** [skade/rust-three-days-course#91]
# Overview

* [ ] Go relationship is interesting
* [x] For "Development" we might want to talk about "Things are not complete when there is nothing left to add, but when there is nothing left to take away."
* [x]  For "Release Method" need to say "Semantic Versioning" instead of "SemVer"
* [x] For "Goals" we should say "Programming in the large" instead of "Large projects"
* [x] For "Rust wants to be usable first" explain that PRs/issues to Rustc are easy if there is a cap missing.
* [x] For "Pragmatic" note that even basic strutures (like linked lists) need unsafe.
* [x] For "Language Features" maybe note about "Zero Cost Abstractions"
* [x] For "Where do Rustaceans come from" add Javascript

# Installation

* [x] For "Important Commands" translate to English
* [x] For "Important Commands" Note about override
* [x] For "Once more with Cargo" the `cat hello-world` is wrong
* [x] For "A Little Look Around" link to Cargo manifest doc

# Basic Types

* [x] For casting do `4_i64` instead of `4i64`
* [x] On "Allocation" note about `new()`
* [x] "Tuples" example is wrong.

# Cargo

* [ ] Talk about cargo <-> Package manager interaction.

# Documentation

* [x] Translate "How is it Generated" code snippet
* [x] "Code examples" example is not colored.
* [x] `rust,no_run` example
* [x] Last slide should leave `cargo doc --open`

# Memory COnsiderations

* [x] "Trait Objecs" is missing a T.

# Overall

* [ ] "How to read documentation"
* [x] Rust-docs seems to be included in Rustup

# Feedback

* [ ] More "Where would I use this in the real world" not necessarily code examples.

**Comment by skade** [comment]
@Hoverbear I have two questions here: (possibly more later)

>  Rust-docs seems to be included in Rustup

Well, only if you installed the respective package once. That was changed a while ago.

>  Go relationship is interesting

Is that the relationship with the programming language go? Where would we mention that?

>  "Tuples" example is wrong.

The example currently is:

```rust
fn main() {
    let p = (1, 2);
    println!("{}", p.0);
    println!("{}", p.1);
}
```

What's wrong about it?

**Comment by skade** [comment]
>  For "Pragmatic" note that even basic strutures (like linked lists) need unsafe.

I'm not sure I agree. That's in the language overview. Also, linked lists are not a basic structure.

**Comment by skade** [comment]
>  For "Rust wants to be usable first" explain that PRs/issues to Rustc are easy if there is a cap missing.

I don't agree with that, the approachability of the project (sadly) has nothing to do with the usability of the language. The approachability of the project is covered in the community chapter.

**Comment by skade** [comment]
> Talk about cargo <-> Package manager interaction.

I'm not sure about that, it blows the chapter up more, but I don't see that much of a gain.

**Comment by Hoverbear** [comment]
Rustdoc problem fixed in a4e5b9e118027382c7bd67640065628a07140096, it's already installed as we noted.

**Comment by Hoverbear** [comment]
The majority of these were fixed or disgarded.
